be/src/column/column_helper.h

```diff
@@ -309,6 +313,33 @@ class ColumnHelper {
 
             start_offset += kBatchNums;
         }
+#elif defined(__ARM_NEON__) || defined(__aarch64__)
+        const uint8_t* f_data = filter.data() + from;
+        constexpr size_t data_type_size = sizeof(T);
+
+        constexpr size_t kBatchNums = 128 / (8 * sizeof(uint8_t));
+        while (start_offset + kBatchNums < to) {
+            uint8x16_t filter = vld1q_u8(f_data);
+            if (vmaxvq_u8(filter) == 0) {
+                // skip
+            } else if (vminvq_u8(filter)) {
+                memmove(data + result_offset, data + start_offset, kBatchNums * data_type_size);
+                result_offset += kBatchNums;
+            } else {
+                for (int i = 0; i < kBatchNums; ++i) {
+#ifndef NDEBUG
```

Comment:
why write a different logic when debugging

Reply:
the second arguments for vgetq_lane_u8 should be a const literal. but when we build SR in ASAN. the loop couldn't unrolled and we will get a compile error

Reply:
How about explaining in the code

Reply:
fixed